### PR TITLE
Add colour coding for village prefix

### PR DESCRIPTION
**Pull request by RunjiaChen**

### CODE REVIEW

File: utils/constants.js
Comment (davidgohzk) [inline utils/constants.js:1]: Could you combine all the venue options into a single Javascript object? There is no need for multiple definitions.

2. Styles should be encapsulated into another component, say under `component/TextComponent` maybe can called it ColouredVenue or something like that kind

**Comment by RunjiaChen** [comment]
- Create custom colour from website (village-1, village-2, village-3, etc.)
https://tailwindcss.com/docs/customizing-colors
- style individual options using classnames 
- change the rendertable to tailwind, do not use style
- use format string for dynamic colour change

**Comment by lutfir-cpu** [comment]
- Added #118
- Note: Both tailwind.config.js and constants.js needs to be edited every time we add a new village (tailwind has issues detecting dynamic custom colours in the config file)

File: .vscode/settings.json
Comment (davidgohzk) [inline .vscode/settings.json:1]: Remove this from git history through .gitignore

File: pages/registration/index.js
Comment (davidgohzk) [inline pages/registration/index.js:1]: Are you sure this is update is necessary?

File: pages/records/index.js [hunk 1]
```diff
@@ -50,6 +51,14 @@ function PatientList() {
     setPatientCode(searchValue);
   }
 
+  function handleChangeWithStyle(e) {
```
Comment (davidgohzk) [inline pages/records/index.js:54]: What Change, add description

File: components/records/Header.js [hunk 2]
```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import moment from 'moment';
 import { ChevronDownIcon } from '@heroicons/react/24/outline';
-import { CLOUDINARY_URL } from '@/utils/constants';
+import { CLOUDINARY_URL, villageColorClasses } from '@/utils/constants';
```
Comment (davidgohzk) [inline components/records/Header.js:4]: Where is this villageColorClasses? I dont see it in constants

File: tailwind.config.js
Comment (davidgohzk) [inline tailwind.config.js:1]: Update file to put safelist inot the module.export and use Regex: https://tailwindcss.com/docs/content-configuration

File: components/records/Header.js (hunk 2 above)
Comment (keithxun) [reply components/records/Header.js:4]: uhhh it is inside
image

File: tailwind.config.js
Comment (keithxun) [reply tailwind.config.js:1]: Damn, I didnt know safelist can use regex

**Pull request merged**